CONTRIBUTING.md

```diff
@@ -46,50 +46,60 @@ Here are a few guidelines to follow when submitting a pull request:
 
 Questions or need help with setup? Feel free to open an issue here [https://github.com/18F/web-design-standards/issues](https://github.com/18F/web-design-standards/issues).
 
-### Building the project locally with Gulp
+### Building the project locally with gulp
 
-The Draft Web Design Standards `uswds` package and style guide website are built
-using Gulp automation. To use Gulp, please make sure you have it installed
-on your machine globally.
+The Draft U.S. Web Design Standards `uswds` package (the ZIP download and the
+files needed to use the Standards on your project) and Standards website (our
+public site that displays examples of each component and the HTML code) are
+built using gulp automation. If you’d like to use to use gulp, first make sure
+you've installed it on your machine globally.
 
 ```sh
 npm install --global gulp-cli
 ```
 
-Once you have Gulp installed globally, you can run the following command to
-see all the commands available to you.
+To start, you’ll need to run the following command to
```

Comment:
To tighten things up a bit, I might recommend starting this sentence as follows:

"To start, run the following command to ..."